ambry-api/src/main/java/com.github.ambry/clustermap/ClusterMap.java

```diff
@@ -56,6 +56,14 @@
    */
   byte getLocalDatacenterId();
 
+  /**
+   * Gets the name of the datacenter from the ID.
+   *
+   * @param id the ID of the datacenter
+   * @return name of the datacenter from the ID.
+   */
+  String getDatacenterName(byte id);
```

Comment:
what is this method supposed to return if the datacenter wasn't found?

Reply:
return null. added to javadoc.